sklearn/metrics/tests/test_pairwise.py

```diff
@@ -299,7 +291,7 @@ def test_pairwise_precomputed_non_negative():
 
 
 _minkowski_kwds = {"w": np.arange(1, 5).astype("double", copy=False), "p": 1}
-_wminkowski_kwds = {"w": np.arange(1, 5).astype("double", copy=False), "p": 1}
```

Comment:
I think you can completely remove all the wminkowski stuff

Reply:
Great catch! I have removed the `_minkowski_kwds` variable remove pytests with `marks=pytest.mark.skipif(sp_version >= parse_version("1.6.0")`. 

Parameters for pytests look like this - 
```
@pytest.mark.parametrize(
    "func, metric, kwds",
    [
        (pairwise_distances, "euclidean", {}),
        (pairwise_distances, minkowski, {}),
        (pairwise_distances, "minkowski", {}),
        (pairwise_kernels, "polynomial", {"degree": 1}),
        (pairwise_kernels, callable_rbf_kernel, {"gamma": 0.1}),
    ],
)
```

I do see the value of keeping - 
```
(pairwise_distances, minkowski, {}),
(pairwise_distances, "minkowski", {}),```

Any concerns?

Reply:
I was talking about removing `wminkowski` (starting with a `w`) but you removed also something about `minkowski` (starting with a `m`). Can you undo the changes you did related to `minkowski` (starting with a `m`)?

For more context, `wminkowski` has been deprecated in scipy 1.6, see [doc](https://docs.scipy.org/doc/scipy-1.6.3/reference/generated/scipy.spatial.distance.wminkowski.html?highlight=wminkowski#scipy.spatial.distance.wminkowski)

Reply:
added back `_minkowski_kwds`.